docs/source/libraries.rst

```diff
@@ -490,6 +490,60 @@ original signature, thus blinding type checkers and other tools that
 provide signature assistance. As such, library authors are discouraged
 from creating decorators that mutate function signatures in this manner.
 
+Aliasing Decorators
+-------------------
+
+When writing a library with a couple of decorator factories 
+(i.e. functions returning decorators, like ``complex_decorator`` from 
+:ref:`annotating-decorators` section) it may be tempting to create a shortcut 
+for a decorator. 
+
+Different type checkers handle ``TypeAlias`` involving ``Callable`` in a 
+different manner, so the most portable and easy way to create a shortcut 
+is to define a callable ``Protocol`` as described in `PEP 
+544 <https://peps.python.org/pep-0544/#callback-protocols>`_.
+
+There is already a ``Protcol`` called ``IndentityFunction`` defined in ``_typeshed``:
```

Comment:
```suggestion
There is already a ``Protocol`` called ``IdentityFunction`` defined in ``_typeshed``:
```

It might also be worth linking to the `_typeshed` README here: https://github.com/python/typeshed/blob/master/stdlib/_typeshed/README.md